op-e2e/setup.go

```diff
@@ -562,28 +562,21 @@ func (cfg SystemConfig) Start(t *testing.T, _opts ...SystemConfigOption) (*Syste
 		}
 	}
 
+	// [Kroma: START]
```

Comment:
Why did these change?

Reply:
As mentioned in description of this PR, the challenger was unable to get the trace because the L2 node settings were not allowed to call `kroma_getBlockTraceByNumberOrHash`.